Session-5/CurrencyExchangeApplication/CurrencyConverter/CurrencyConverter.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using RestSharp;
+using Newtonsoft.Json;
+using System.Configuration;
+
+namespace CurrencyExchange
+{
+    public class CurrencyConverter
+    {
+        private static List<ExchangeRateDBO> GetApidata()
+        {
+            RestClient client = new RestClient("https://api.apilayer.com/currency_data/live?source=INR&currencies=");
+            RestRequest request = new RestRequest();
+            request.AddHeader("apikey", "NKJlyNE9umLraoNvP87N7W9v6NNePbI8");
+            ApiResponse apiResponse = JsonConvert.DeserializeObject<ApiResponse>(client.Get(request).Content);
+            List<ExchangeRateDBO> dbRates = apiResponse.quotes.Select(x => new ExchangeRateDBO(x.Key, x.Value, DateTime.Now)).ToList();
+            return dbRates;
+        }
+
+        public static double ConvertCurrency(string fromCurrency, string toCurrency, double currencyValue)
+        {
+            if (fromCurrency.Equals(toCurrency))
+            {
+                return currencyValue;
+            }
+            CurrencyExchangeDB DbObject = new CurrencyExchangeDB();
```

Comment:
Fix naming convention for the variable. Also rename the variable as exchangeRateDB.
Pass connection string in constructor.